js/alert.js

```diff
@@ -0,0 +1,17 @@
+const showAlert = (message)=>{
```

Comment:
В целом промисы тут только для примера был, по идеи он не нужен, ну и можешь сразу чуть чуть стилей добавить... Даже если не будешь это использовать, чтобы всегда работало как нужно
```suggestion
const showAlert = (message) => {
  const el = document.createElement('div');
  el.textContent = message;
  el.style.background = 'red';
  el.style.position = 'fixed';
  el.style.top = 0;
  el.style.right = 0;
  el.style.left = 0;
  el.style.padding = '15px';
  el.style.textAlign = 'center';

  document.body.appendChild(el);

  return new Promise((resolve) => {
    setTimeout(() => {
      document.body.removeChild(el);
      resolve();
    }, 4000);
  });
};
```